app/admin/rush/page.tsx

```diff
@@ -83,15 +112,49 @@ export default function RushEvents() {
                   <Avatar placeholderInitials={event.name[0]} rounded />
                 </div>
                 <div className="min-w-0 flex-1">
-                  <p className="truncate text-sm font-medium text-gray-900 dark:text-white">{event.name}</p>
-                  <p className="truncate text-sm text-gray-500 dark:text-gray-400 mr-1">{formatMongoDate(event.dateCreated)}</p>
-                  <code className="truncate text-sm text-gray-500 dark:text-gray-400">{rushCategoriesCodeToggled[categoryId] ? (`Code: ${event.code}`) : "Code: •••••••"}</code>
+                  <p className="truncate text-m font-medium text-gray-900 dark:text-white">{event.name}</p>
+                  {/* <p className="flex gap-2 truncate text-sm text-gray-500 dark:text-gray-400 mr-1">
+                    Created:
+                    <MongoTimestamp datestring={event.dateCreated} />
+                  </p> */}
+                  <p className="flex gap-2 truncate text-sm text-gray-500 dark:text-gray-400 mr-1">
+                    Last Modified:
+                    <MongoTimestamp datestring={event.lastModified} />
+                  </p>
+                  <p className="flex gap-2 truncate text-sm text-gray-500 dark:text-gray-400 mr-1">
+                    Deadline:
+                    <MongoTimestamp datestring={event.deadline} />
```

Comment:
Maybe just calling the library itself makes more sense - I saw the code for the wrapper and it just essentially does the same / unless we have plans to make specific timestamps, I think replacing it with the library and deleting the mongotimestamp component makes more sense!

Reply:
agreed, I went back and forth on that lol